### PR TITLE
Rework of the Workspace Concept section.

### PR DESCRIPTION
**Pull request by ninjeeter**
Ready for PR.

### CODE REVIEW

File: src/concepts/nodes.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+<img alt="Convert workflow" src="/_images/workflow_convert_basic.png" center/>
+
+## Connecting Nodes
+
+---
+
+A `Connection` is visually represented by the gray line between nodes and determine the order of execution.
+
+- Workflows use a top-down heirachical structure (_the Node at the very top represents the beginning of the flow and the Node at the bottom represents the end of the flow_).
```
Comment (Sytten) [inline src/concepts/nodes.md:19]: I am not a super fan of those single indented dots, but I am not sure what to use instead. Maybe we don't need anything?

File: src/concepts/nodes.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+_Example (pictured above):_
+
+- _The value_ `$start.data` _is the output of the_ **Start** _Node being taken as input by the_ **Base64 Encode** _Node_.
+- _The output of the_ **Base64 Encode** _Node will be referenced by the_ **End** _Node as_ `$base64_encode.data`.
+
+## Node Categories
+
+---
+
+- **Encoding/Decoding/Hashing/Dehashing Nodes** (Blue) - Base64/Hex/HTML/JWT/MD5/SHA1/SHA2/URL
```
Comment (Sytten) [inline src/concepts/nodes.md:67]: I prefer we something along:
- **Start/Stop Nodes** (Yellow)
- **Control Nodes** (Green)
  Like `If/Else`, this allows you to control the execution flow (ref execution flow) 
- **Code Nodes** (Red)
  Like `Shell`, `JS`, those allow you to... 
- **All Other Nodes** (Blue)

Also the list here is only for convert workflows, it is different right now for active/passive. We should mention that each type has different nodes based on its capabilities and that we are expanding them (https://github.com/caido/caido/issues?q=is%3Aopen+is%3Aissue+label%3A%22topic%3A+workflow%22).

File: src/concepts/workflows.md [hunk 3]
```diff
@@ -1,82 +1,27 @@
...
-<img alt="Base64 Node" src="/_images/workflow_convert_node_base64.png" width="500" center/>
-
-Caido comes with a default list of nodes. Here are a few we support:
-
-- Encoding nodes (Base64/URL/Hex)
-- Hashing nodes (SHA1/SHA2)
-- Code nodes (Shell/JavaScript)
-- Control nodes (If-Else/If-Else-Javascript)
-- Misc nodes (Set-Color/HTTPQL)
+**Workflows** are the main way to customize Caido to suit your needs. Think of Workflows as custom shortcuts that will perform an action/a sequence of actions under certain specified conditions. These "shortcuts" can be saved as a Workflow in Caido so they can be utilized repeatedly.
```
Comment (Sytten) [inline src/concepts/workflows.md:7]: It is not the main way anymore, its A way to customize caido with tasks executed on the server side (ref to client/server architecture).
Unsure about the `shortcuts` analogy, we generally define them as building your own mini-plugins by combining multiple actions and scripts together (`that will perform an action/a sequence of actions under certain specified conditions` that is good).

File: src/concepts/workflows.md [hunk 4]
```diff
@@ -1,82 +1,27 @@
...
-<img alt="Workflow Node Inputs" src="/_images/workflow_node_inputs.png" width="400" center/>
+---
 
-When using a constant value, executing the node will use the value as-is.
-When using a reference value, executing the node will use the output of a previous node.
+There are three main types of Workflows, each of which are applied differently:
 
-To use reference values, the content of the text field should be under the format:
+1. `Passive Workflows` - will automatically trigger based on the conditions specified when creating the Workflow.
+2. `Active Workflows` -  must be manually triggered and meet the conditions specified during creation.
```
Comment (Sytten) [inline src/concepts/workflows.md:20]: Unsure what you meant by `conditions specified` for active?

File: src/concepts/workflows.md [hunk 5]
```diff
@@ -1,82 +1,27 @@
...
+3. `Convert Workflows` - take strings as input, perform the specified modifications and then outputs the end result.
```
Comment (Sytten) [inline src/concepts/workflows.md:21]: bytes is more appropriate

**Review by Sytten — CHANGES_REQUESTED** [review]
Small changes, good job :)

File: src/concepts/workflows.md (hunk 4 above)
Comment (ninjeeter) [reply src/concepts/workflows.md:20]: Referring to the start/end nodes but I'll think of some other terminology.

File: src/concepts/nodes.md (hunk 1 above)
Comment (ninjeeter) [reply src/concepts/nodes.md:19]: I think for the structure there should be something to make it easier on the eyes to parse through. Also helps to delineate subsets of information pertaining to a parent. Maybe just a differently shaped bullet point?

Comment (ninjeeter) [reply src/concepts/nodes.md:19]: Ah wait, misread. Yeah I agree. Will remove it.

**Review by Sytten — APPROVED** [review]
Hurray 🎉

**Pull request merged**